include/NovelRT/NovelRunner.h

```diff
@@ -14,7 +14,13 @@ namespace NovelRT {
   class NovelRunner {
 
   public:
+    /**
+     * An event that occurs when the scene has to be (re)constructed.
+     */
     Utilities::Event<> SceneConstructionRequested;
+    /**
+     * An event that occurs on a frame update, with the delta Timestamp between the last frame update and the new one.
```

Comment:
You could probably add on here, `Use this to define game behaviour that is required to be done per-frame.`, just to make it completely clear.

Reply:
Will add 👌